.github/workflows/ci.yml

```diff
@@ -111,14 +97,23 @@ jobs:
           S3_ENDPOINT: https://s3.sbg.cloud.ovh.net/
           S3_REGION: sbg
         run: |
+          if [ "$TEST_DB_BACKEND" == "postgres" ]; then
+            export QUETZ_TEST_DATABASE="postgresql://postgres:mysecretpassword@${POSTGRES_HOST}:${POSTGRES_PORT}/postgres"
+            echo "Running with postgres"
+          fi
+
           # add micromamba to path
           export PATH=$(dirname $MAMBA_EXE):$PATH
           echo "adding micromamba to path: $MAMBA_EXE"
 
           pip install --no-deps git+https://git@github.com/regro/libcflib@master
           
-          quetz plugin install ./plugins/${{ matrix.plugin}}
-          pytest -v ./plugins/${{ matrix.plugin }}
+          for f in ./plugins/quetz_*
+          do
+            echo "Testing plugin" $f
```

Comment:
I think pytest might do some of this automatically, but there are fancy ways of formatting in github actoins with e.g. we could group log lines with this https://docs.github.com/en/actions/using-workflows/workflow-commands-for-github-actions#grouping-log-lines

E.g. using `echo ::group::$f` and then `echo ::endgroup::` at the end

Reply:
Thanks for the suggestion